source/extensions/filters/network/http_connection_manager/config.h

```diff
@@ -99,12 +100,12 @@ class InternalAddressConfig : public Http::InternalAddressConfig {
       return unix_sockets_;
     }
 
-    // TODO(snowp): Make internal subnets configurable.
-    return Network::Utility::isInternalAddress(address);
+    return cidr_ranges_.contains(address) || Network::Utility::isInternalAddress(address);
```

Comment:
I think that `isInternalAddress()` should not be used if the user configured cidr ranges here. So basically if it's unconfigured, use the defaults (10.0.0.0/8 et al), and if it is configured, only use the configured cidrs.

Reply:
fixed, PTAL. Thanks!